lib/sqlalchemy/sql/elements.py

```diff
@@ -4630,20 +4768,22 @@ def __init__(
 
         self.is_literal = is_literal
 
-    def get_children(self, *, column_tables=False, **kw):
+    def get_children(
+        self, *, column_tables: bool = False, **kw: Any
+    ) -> List[Any]:
         # override base get_children() to not return the Table
         # or selectable that is parent to this column.  Traversals
         # expect the columns of tables and subqueries to be leaf nodes.
         return []
 
     @property
-    def entity_namespace(self):
+    def entity_namespace(self) -> Union[_EntityNamespace, NoReturn]:
         if self.table is not None:
             return self.table.entity_namespace
         else:
             return super().entity_namespace
 
-    def _clone(self, detect_subquery_cols=False, **kw):
+    def _clone(self, detect_subquery_cols: bool = False, **kw: Any) -> Any:
```

Comment:
Calls `kw.pop` while `**kw: Any`, thus returns `Any`. All other `_clone()` cases have `**kw` annotated as `Any` too - no idea what it acually accepts. Therefore, I've left `Any` on the return, can't resolve any further.

Reply:
The method is private, so it's not intended to be used anywhere but withing the class itself (which seriously reduces the possibility of unintended usage), apart from that the whole typing effort is intended primarily for the public-facing API, so it's fine to just use `Any` in this case.